.circleci/config.yml

```diff
@@ -3,24 +3,15 @@ version: 2.1
 executors:
   nodejs:
     docker:
-      - image: circleci/node:10
+      - image: dannycoates/fxa-circleci
 commands:
   npm-install:
     parameters:
       module:
         type: string
         default: "dependencies"
     steps:
-      - restore_cache:
-          keys:
-            - v2-<< parameters.module >>-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - v2-<< parameters.module >>-
       - run: npm ci
-      - save_cache:
-          paths:
-            - node_modules
-          key: v2-<< parameters.module >>-{{ checksum "package.json" }}
```

Comment:
these caches weren't really doing anything

Reply:
When I was watching one of the runs, I noticed their network is so slow, that it can be faster to do some work from scratch rather than wait for large cache stores to move between wherever they're moving between.